doc/source/config.rst

```diff
@@ -106,6 +106,19 @@ It can either be ``enabled`` or ``disabled``.
 
 .. _config_global:
 
+Prevent sloppy comparison
+^^^^^^^^^^^^^^^^
+
+sloppy_comparison, disabled by default, will prevent php type
+juggling (``==``) , preventing any bypass of a comparison.
```

Comment:
It might be nice to add a reference to the [documentation]( https://secure.php.net/manual/en/language.types.type-juggling.php)